ofl/pacifico/METADATA.pb

```diff
@@ -1,5 +1,5 @@
 name: "Pacifico"
-designer: "Vernon Adams, Jacques Le Bailly, Botjo Nikoltchev, Ani Petrova"
+designer: "Vernon Adams, Jacques Le Bailly, Botjo Nikoltchev, Ani Petrova, Raphael Alegbeleye"
```

Comment:
It would be best to move Raphael to the third place (after Jacques) given him will have bio info while Botjo and Ani don't have it.

Reply:
yes good idea!